Directory.Build.props

```diff
@@ -4,9 +4,17 @@
     <Authors>Microsoft.Toolkit</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <NoPackageAnalysis>true</NoPackageAnalysis>
-    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</PackageLicenseUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+<package>
+  <metadata>
+    <icon>images\nuget.png</icon>
+    <license type="file">License.md</license>
+  </metadata>
+  <files>
+    <file src="WindowsCommunityToolkit\build\nuget.png" target="images\" />
+    <file src="WindowsCommunityToolkit\License.md" target="" />
+  </files>
```

Comment:
These files should be added like the `stylecop.json` is below. [Or see the 2nd example of packing a file here](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).

We're also at the root already, so the paths should just be `build\nuget.png` and `license.md`